src/neo/VM/Helper.cs

```diff
@@ -298,6 +299,7 @@ public static StackItem ToStackItem(this ContractParameter parameter)
 
         private static StackItem ToStackItem(ContractParameter parameter, List<(StackItem, ContractParameter)> context)
         {
+            if (parameter is null) return StackItem.Null;
```

Comment:
Here should be `parameter.Value`
![image](https://user-images.githubusercontent.com/43694095/75140912-ce52d780-572a-11ea-9523-807a80a44e81.png)

Reply:
Interop it's null, and must rise the exception.

Reply:
There isn't `case ContractParameterType.Any`, so when the `parameter`'s type is Any and value is null, it will still throw an ArgumentException.

Reply:
I think that it should throw the exception

Reply:
So now this pr doesn't fix #1416 ? Because when I call `GetApplicationLog` method of `RpcClient` module, the State of Notifications[0] is still null.

Reply:
now, it should be `StackItem.Null`, could you check it please?

Reply:
I have changed another [PR](https://github.com/neo-project/neo-modules/pull/171) for this issue, please help to check.